core-header-panel.css

```diff
@@ -31,6 +31,11 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
   -webkit-overflow-scrolling: touch;
 }
 
+#mainContainer {
```

Comment:
There's a `#mainContainer` selector right above this one. Can they be collapsed into one?